cmake/platform/settings.cmake

```diff
@@ -0,0 +1,4 @@
+# set installation prefix
+if( PREFIX )
+  set(CMAKE_INSTALL_PREFIX "${PREFIX}")
```

Comment:
Anyway, like DDuarte said, this file (along with a couple of others) don't belong in this PR.
See if you can find a way to separate your PR files from the normal 6.x updates.